content/en/community/contributing/code/creating-good-first-issues.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Creating good first issues"
+linkTitle: "Good first issues"
+weight: 400
+description: >
+  How to set community members up for success with good first issue tickets
+
+---
+
+
+The goal of these tickets is to make it really easy for a first time contributor!  Yes, it might mean that the time  spent writing out the good first issue  could have been put toward already completing the issue. If true,  this is a sign you're doing the right work ;)
+
+1. Find an issue you know a lot about.  Ideally one where you know exactly what the steps are
+2. Edit the body of the issue and add a `## Good first issue` section
```

Comment:
We miss a step for adding the "Good First Issue" label. Additionally, I would like to point out [the difference with "Help wanted" tickets](http://localhost:1313/community/contributing/first-time-contributors/#your-first-issue).

Reply:
:facepalm: - of course!

fixed in 98152648